stack.yaml

```diff
@@ -66,17 +66,17 @@ extra-deps:
 - streaming-binary-0.3.0.1
 - generic-monoid-0.1.0.0
 - canonical-json-0.6.0.0
-- cborg-0.2.2.0
 - statistics-linreg-0.3
 - network-3.1.0.1
 - connection-0.3.1
 - Unique-0.4.7.6
 - moo-1.2
 - gray-code-0.3.1
 
-# - bimap-0.4.0
-# - hedgehog-1.0
-# - tasty-hedgehog-1.0.0.1
+- git: https://github.com/well-typed/cborg
```

Comment:
Is this a fix for `pprint: Unexpected token` or something else?

Reply:
It is indeed.